src/exchanges/fetch.ts

```diff
@@ -99,6 +97,13 @@ const createFetchSource = (operation: Operation) => {
         abortController !== undefined ? abortController.signal : undefined,
     };
 
+    if (fetchOptions.method === 'GET' && operation.operationName === 'query') {
```

Comment:
I'd go the ternary route here or pull fetchOptions parsing into a function with a return. That way fetchOptions doesn't change after assignment